### ISSUE TITLE
azul.pyd not working on win7 and win11

### ISSUE THREAD

**Issue by chenyulue** [fschutt/azul#360]
I downloaded the [azl.pyd](https://github.com/fschutt/azul/releases), and followed the [instruction](https://azul.rs/guide/1.0.0-alpha1/Installation) to set up the same project directory. When I ran the first example (the counter) shown at https://azul.rs/, the gui app didn't work correctly as the following image shows. The os is win11, and the python version is python 3.11.6.

![2023-11-08_173946](https://github.com/fschutt/azul/assets/44137770/7817bb76-c08e-41c7-8e68-a5ad70498c24)

The same issue happens on win7.

Am I missing anything with respect to the usage of azul from python?

**Comment by fschutt** [comment]
Yeah I'm not sure what exactly happens, but the azul.pyd is a bit out of date. I believe there was a redrawing bug that doesn't properly adjust the DPI on the first frame. Not sure.
